### PR TITLE
fix(browser): orchestrator preventing iframe mouse interaction

### PR DESCRIPTION
**Pull request by userquin**
### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. If the feature is substantial or introduces breaking changes without a discussion, PR might be closed.
- [ ] Ideally, include a test that fails without this PR but passes with it.
- [ ] Please, don't make changes to `pnpm-lock.yaml` unless you introduce a new test example.

### Tests
- [ ] Run the tests with `pnpm test:ci`.

### Documentation
- [ ] If you introduce new functionality, document it. You can run documentation with `pnpm run docs` command.

### Changesets
- [ ] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.

### CODE REVIEW

**Comment by userquin** [comment]
Weird behavior when resizing the panel.

File: packages/browser/src/client/orchestrator.ts [hunk 1]
```diff
@@ -152,7 +151,7 @@ async function createTesters(testFiles: string[]) {
   const container = await getContainer(config)
 
   if (config.browser.ui) {
-    container.className = ''
+    // container.className = ''
```
Comment (userquin) [inline packages/browser/src/client/orchestrator.ts:154]: remove this comment

File: packages/ui/client/components/BrowserIframe.vue [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <script setup lang="ts">
+import { registerResizeListener } from '~/composables/client/resizing'
+
 const viewport = ref('custom')
+const resize = ref(false)
```
Comment (userquin) [inline packages/ui/client/components/BrowserIframe.vue:5]: rename this to `resizing`

File: packages/ui/client/composables/client/resizing.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  }
+
+  function notify(active: boolean) {
+    for (const listener of listeners)
+      listener(active)
+  }
+
+  provide(resizingSymbol, addResizeListener)
+
+  return { notify }
```
Comment (userquin) [inline packages/ui/client/composables/client/resizing.ts:23]: rename this to `notifyResizing`

File: packages/ui/client/components/BrowserIframe.vue [hunk 4]
```diff
@@ -61,30 +70,54 @@ function changeViewport(name: string) {
...
     </div>
   </div>
 </template>
 
 <style>
+#tester-ui.resizing iframe {
+  pointer-events: none;
+}
+[data-viewport="custom"] {
+  padding: 11px;
```
Comment (sheremet-va) [inline packages/ui/client/components/BrowserIframe.vue:91]: why do you need margins and paddings?

File: packages/browser/src/client/orchestrator.ts [hunk 5]
```diff
@@ -151,7 +151,7 @@ async function createTesters(testFiles: string[]) {
...
```
Comment (sheremet-va) [inline packages/browser/src/client/orchestrator.ts]: why did you comment that out?

**Review by sheremet-va — CHANGES_REQUESTED** [review]

File: packages/ui/client/components/BrowserIframe.vue (hunk 4 above)
Comment (sheremet-va) [reply packages/ui/client/components/BrowserIframe.vue:91]: I intentionally _removed_ all paddings

File: packages/browser/src/client/orchestrator.ts [hunk 6]
```diff
@@ -151,10 +150,8 @@ async function createTesters(testFiles: string[]) {
   const config = getConfig()
   const container = await getContainer(config)
 
-  if (config.browser.ui) {
-    container.className = ''
+  if (config.browser.ui)
     container.textContent = ''
```
Comment (sheremet-va) [inline packages/browser/src/client/orchestrator.ts:154]: Why did you remove this? Only the fallback text should be centered

Comment (userquin) [reply packages/browser/src/client/orchestrator.ts:154]: now it is always centered, the container and the iframe will have the same size

Comment (userquin) [reply packages/browser/src/client/orchestrator.ts:154]: I've removed also the overflow auto in the container and so centering the content doesn't matter

Comment (sheremet-va) [reply packages/browser/src/client/orchestrator.ts:154]: the iframe **should no** be centered

File: packages/ui/client/components/BrowserIframe.vue (hunk 4 above)
Comment (userquin) [reply packages/ui/client/components/BrowserIframe.vue:91]: It is about the 11px on the left resizer (avoid overlap)

Comment (userquin) [reply packages/ui/client/components/BrowserIframe.vue:91]: I can remove both margin and padding, shouldn't be a problem

File: packages/browser/src/client/orchestrator.ts (hunk 6 above)
Comment (sheremet-va) [reply packages/browser/src/client/orchestrator.ts:154]: Can you please explain why you did it and not just state that you did it? I can see the code, I don't see _why_. The iframe was never meant to be centered

**Comment by sheremet-va** [comment]
There is a lot of changes to styles here that have nothing to do with resizing. Just keep the resizng bug fix please

File: packages/ui/client/components/BrowserIframe.vue (hunk 4 above)
Comment (sheremet-va) [reply packages/ui/client/components/BrowserIframe.vue:91]: There is no overlap, the resizer appears on top of the panel
